docs/1-tendermint-overview.md

```diff
@@ -0,0 +1,92 @@
+If a correct process receives more than 2/3 Precommit messages for a null value in the same round, it moves to the next round.
+
+After a decision has been made, processes continue to agree on other values on another height (consensus instance) and they repeat the process described above in order to agree on different transactions.
+
+In order to ensure that processes will eventually come to a decision, there are some constraints that are applied. These rules are used to prevent any malicious attempt to cause more than one block to be committed at a given height. 
+When a validator sends a Precommit for a value *v* at round *r*, we say it is *locked* on *v*. Validators can propose and send Prevote messages for a value they have locked and they can only change the locked value if they receive a more recent proposal with a quorum of Prevote messages. 
+
+These conditions ensure both the safety and liveness properties of the consensus algorithm because a validator cannot send a Precommit message without sufficient evidence and cannot send a Precommit message for a different value at the same time.
+
+Tendermint is notable for its simplicity and performance and is currently one of the best consensus algorithm used in production.
+
+## Properties of consensus 
+
+The Tendermint consensus algorithm guarantees the following properties:
+
+- **Agreement**: No two correct processes decide on different values
+
+- **Validity**: A value decided is a valid value, respect to the specifications of validity of a message
+
+- **Termination**: All correct processes eventually decide
+
+- **Integrity**: No correct process decides more than once in a single height 
+
+We limit the discussion to the agreement property only because its proof can be useful to better understand the fork accountability problem. 
+We invite interested readers to read the official paper for a more formal, complete and rigorous proof of all the properties listed above.
+
+## Agreement property
+
+In this section we'll give a simple proof of the agreement property for the Tendermint algorithm and show why the majority assumption is important for ensuring this property.
+
+The key idea behind the proof of the agreement property is that any two sets of *2f + 1* processes have at least one correct process in common respect to the Tendermint context.
+
+We know that Tendermint assumes that *n > 3f*, for simplicity we can simplify it such that:
+ 
+    n = 3f + 1
+
+We have two sets of *2f + 1* processes, their sum can be written as:
+ 
+    2(2f + 1) = 4f + 2 = 3f + 1 + f + 1 = n + f + 1
+     
+That means that the intersection of these two sets contain at least *f + 1* processes, therefore at least one correct process.  
+
+Given this result, it's quite easy to understand that, if there are at most *f* faulty processes, it's not possible to violate the agreement property in the Tendermint consensus algorithm, as we'll show in the next section.
```

Comment:
Contractions. I will not add any more comments regarding that, but I think we should remove them throughout the entire document.

Reply:
I agree contractions do not sound formal at all and now they are (hopefully) not present anymore. However, I didn't think about making a formal discussion while writing the documentation (for example, rhetorical questions are used and they are not formal). I just wanted to explain the problem and its context using simple words and common language in order to make other readers who don't know the subject understand the project. If we want to use a different language (not a problem for me), I might need to change the style of all documents but I'm not sure this is what we wanted. Maybe we can discuss this tomorrow